.circleci/config.yml

```
@@ -81,7 +81,7 @@ jobs:
             - ./image.tar
   githubPagesTest:
     docker:
-      - image: mhart/alpine-node:14
+      - image: node:18
```

Looks like we still need apk for rsync.

```suggestion
      - image: node:18-alpine
```